packages/components/tree-v2/src/virtual-tree.ts

```diff
@@ -0,0 +1,165 @@
+import { buildProp, definePropType, mutable } from '@element-plus/utils/props'
+import type {
+  TreeNode,
+  TreeKey,
+  TreeData,
+  TreeOptionProps,
+  FilterMethod,
+} from './tree.type'
+
+// constants
+export const ROOT_TREE_INJECTION_KEY = 'RootTree'
```

Comment:
We can use `InjectionKey`.
```suggestion
export const ROOT_TREE_INJECTION_KEY: InjectionKey<RootTreeType> = Symbol()
```